odbc/2.ibmi.md

```diff
@@ -0,0 +1,96 @@
+
+**`odbcinst.ini`**
+```ini
+[IBM i Access ODBC Driver]
+Description = IBM i Access ODBC Driver
+Driver      = /QOpenSys/pkgs/lib/libcwbodbc.so
+```
+(Note: You don't have to align the `=` signs, it just looks cleaner.)
+
+When you define datasources in `odbc.ini` and `.odbc.ini` (covered below), you will use the name in the `[]` brackets as the name of the driver to use.
+
+If you have multiple drivers to define, you can simply create mutiple entries in the same file, as long as each driver is headed with a driver name in brackets, and no two drivers have the same name.
+
+**`odbc.ini` and `.odbc.ini`**
+
+When you have a driver defined, you will want to also define a datasource name, or DSN. A DSN allows you to define a connection and all of your connection options in one place. When you pass a connection string to an ODBC connection, you simply need to pass the name of the DSN and all of the connection options defined with the DSN will also be included.
+
+DSNs are defined in `odbc.ini` and `.odbc.ini` (note the `.` preceding the latter). These two files have the same structure, but have one important difference: 
+
+* `odbc.ini` can be found in the IFS in `/QOpenSys/etc` and defines DSNs that are available to **all users on the system**. If there are DSNs that should be available to everyone, they can be defined and shared here
+
+* `.odbc.ini` is found in your home directory (`~/`) and defines DSNs that are available **only to you**. If you are going to define DSNs with your personal username and password, this is the place to do it.
+
+The files define DSNs similar to how `odbcinst.ini` defines drivers: You name your DSN with `[]` brackets, then specify keywords and values below them. This is an example of a  stored in `~/.odbc.ini` used to connect to an IBM i with private credentials:
+
+```ini
+[MYDSN]
+Description            = oss73dev IBM i system
+Driver                 = IBM i Access ODBC Driver
+System                 = OSS73DEV.RCH.STGLABS.IBM.COM 
+UserID                 = MYNAME
+Password               = password123
+Naming                 = 0
+DefaultLibraries       = MIRISH
+TrueAutoCommit         = 1
+```
+(**Note: The name of the driver specified in the `Driver` keyword must match the name of a driver defined in `odbcinst.ini`).
+
+Additionally, the `ibm-iaccess.rpm` will automatically create a DSN called `[*LOCAL]` in your `odbc.ini`. When using this DSN, the user credentials used will be `*CURRENT`, which is the user who is running the process that is trying to connect to the ODBC driver.
+
+A complete list of IBM i Access ODBC Driver connection options for both DSN definitions and connection strings can be found at the [IBM Knowledge Center: Connection string keywords webpage](https://www.ibm.com/support/knowledgecenter/ssw_ibm_i_73/rzaik/connectkeywords.htm).
```

Comment:
I'd recommend changing to the 7.4 page, since it will last longer than the 7.3 page.

"ssw_ibm_i_73" -> "ssw_ibm_i_74"